README.md

```diff
@@ -66,17 +60,17 @@ jupyter enterprisegateway --help-all
 jupyter enterprisegateway
 ```
 
-Please check the [Configuration Options page](https://jupyter-enterprise-gateway.readthedocs.io/en/latest/config-options.html) 
+Please check the [Configuration Options page](https://jupyter-enterprise-gateway.readthedocs.io/en/latest/operators/index.html) 
```

Comment:
I think referencing the actual configuration section in the Operators Guide might be more appropriate here.
```suggestion
Please check the [configuration options within the Operators Guide](https://jupyter-enterprise-gateway.readthedocs.io/en/latest/operators/index.html#configuring-enterprise-gateway) 
```